cardano-node/README.md

```diff
@@ -244,6 +244,16 @@ options.
 
 ## Transactions
 
+### Creation
+
+Transactions can be created via the  `issue-genesis-utxo-expenditure` & `issue-utxo-expenditure` commands.
```

Comment:
Can these only spend genesis utxos? Is there a general sub command to generate a plain tx?